totalopenstation/output/tops_xml.py

```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+# filename: tops_xml.py
```

Comment:
I think this file should be named `tops_landxml`? And the same for the tests

Reply:
Ok done.